containers/Investment/RevolvingPool/View/PoolOverview.tsx

```diff
@@ -1,53 +1,192 @@
 import * as React from 'react'
 import { Box, Heading, Table, TableBody, TableRow, TableCell } from 'grommet'
-import { Pool } from '../../../../config'
+import { connect, useSelector } from 'react-redux'
+import { loadPool, PoolDataV3, PoolState } from '../../../../ducks/pool'
+import { toPrecision } from '../../../../utils/toPrecision'
+import { addThousandsSeparators } from '../../../../utils/addThousandsSeparators'
+import { baseToDisplay, feeToInterestRate } from '@centrifuge/tinlake-js'
+import { TINRatioBar } from '../../../../components/TINRatioBar/index'
+import {
+  TokenLogo,
+  BalanceSheetDiagram,
+  BalanceSheetDiagramLeft,
+  BalanceSheetMidLine,
+  BalanceSheetDiagramRight,
+  DividerTop,
+  DividerBottom,
+  DividerInner,
+} from './styles'
+import BN from 'bn.js'
 
-interface Props {
-  pool: Pool
-}
+interface Props {}
 
 const PoolOverview: React.FC<Props> = () => {
+  const pool = useSelector<any, PoolState>((state) => state.pool)
+
+  const poolValue =
+    (pool.data !== null &&
+      (pool.data as PoolDataV3).netAssetValue &&
+      (pool.data as PoolDataV3).reserve &&
+      (pool.data as PoolDataV3).netAssetValue.add((pool.data as PoolDataV3).reserve)) ||
+    '0'
+  const dropRate = (pool.data && pool.data.senior && pool.data.senior.interestRate) || '0'
+
+  const dropTotalValue =
+    pool.data &&
+    (pool.data as PoolDataV3).senior &&
+    (pool.data as PoolDataV3).senior!.totalSupply.mul((pool.data as PoolDataV3).senior!.tokenPrice)
+
+  const tinTotalValue =
+    pool.data && (pool.data as PoolDataV3).junior.totalSupply.mul((pool.data as PoolDataV3).junior.tokenPrice)
+
+  const ratioBase = new BN(10).pow(new BN(20))
+  const currentJuniorRatio = pool.data ? pool.data.currentJuniorRatio.div(ratioBase).toNumber() / 10 ** 7 : 0
+  const minJuniorRatio = pool.data ? pool.data.minJuniorRatio.div(ratioBase).toNumber() / 10 ** 7 : 0
+  const maxJuniorRatio = pool.data ? (pool.data as PoolDataV3).maxJuniorRatio.div(ratioBase).toNumber() / 10 ** 7 : 0
```

Comment:
I think a generic helper is hard, because it really differs per value, but for these three specific items I've added a `parseRatio(num: BN): number)` helper